### PR TITLE
fix(translations): Update ru translations

### PR DESCRIPTION
**Pull request by Altizo**
Fixes to improve interaction.

- fixed logical inconsistencies
- typos
- fixed machine translation errors

### CODE REVIEW

**Comment by marination** [comment]
@Altizo Thanks for continuing to contribute! In the future I'd suggest to make sure your commits are following the right semantics.
Here's a resource for the same: https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716

Commits like `chore: Update ru.csv` or `fix: incorrect translation` are appropriate.

File: erpnext/translations/ru.csv [hunk 1]
```diff
@@ -921,15 +921,15 @@ Employee,Сотрудник,
...
 Employee ID,ID сотрудника,
 Employee Lifecycle,Жизненный цикл сотрудников,
 Employee Name,Имя сотрудника,
 Employee Promotion cannot be submitted before Promotion Date ,Повышение сотрудника не может быть выполнено до даты приступления к должности,
 Employee Referral,Перечень сотрудников,
 Employee Transfer cannot be submitted before Transfer Date ,Передача сотрудника не может быть отправлена до даты передачи,
-Employee cannot report to himself.,Сотрудник не может сообщить себе.,
-Employee relieved on {0} must be set as 'Left',"Сотрудник освобожден от {0} должен быть установлен как ""левые""",
+Employee cannot report to himself.,Сотрудник не может сообщить самому себе.,
+Employee relieved on {0} must be set as 'Left',"Сотрудник освобожден от {0} должен быть установлен как ""Покинул""",
```
Comment (marination) [inline erpnext/translations/ru.csv]:
```suggestion
Employee relieved on {0} must be set as 'Left',"Сотрудник освобожден от {0} должен быть установлен как 'Покинул'",
```

Comment (Altizo) [reply erpnext/translations/ru.csv]: Thank you, I'll fix such errors in the future

Comment (ankush) [reply erpnext/translations/ru.csv]: @Altizo also if possible please validate CSV syntax and if it's loading correctly on UI from your end. (this and future contributions)

ref: https://github.com/frappe/frappe/pull/16769 &  https://github.com/frappe/frappe/pull/16894

**Review by marination — APPROVED** [review]

**Pull request merged**